### ISSUE TITLE
How to fuse batchnorm to conv2d in the graph exported by torch.export

### ISSUE THREAD

**Issue by TingfengTang** [pytorch/pytorch#141116]
I used the torch.export to export my CNN model in eval mode,but the op batchnorm still exists. how to eliminate it. Is there some options in torch.export.export function or I should write a fusion pass by myself. 
Thanks.
code:
```
import torch
import torch.nn as nn
class CNN(nn.Module):
    def __init__(self):
        super().__init__()
        self.conv1 = nn.Conv2d(in_channels=16, out_channels=16, kernel_size=3, stride=1, padding=1)
        self.bn1 = nn.BatchNorm2d(16)

    def forward(self, x):
        x = self.conv1(x)   
        x = self.bn1(x)
        return x

torch.manual_seed(0)
model=CNN().eval()
input=torch.randn(3,16,224,224)
ep=torch.export.export(model,(input,))
print(ep.graph)
```
graph:
```
graph():
    %p_conv1_weight : [num_users=1] = placeholder[target=p_conv1_weight]
    %p_conv1_bias : [num_users=1] = placeholder[target=p_conv1_bias]
    %p_bn1_weight : [num_users=1] = placeholder[target=p_bn1_weight]
    %p_bn1_bias : [num_users=1] = placeholder[target=p_bn1_bias]
    %b_bn1_running_mean : [num_users=1] = placeholder[target=b_bn1_running_mean]
    %b_bn1_running_var : [num_users=1] = placeholder[target=b_bn1_running_var]
    %b_bn1_num_batches_tracked : [num_users=0] = placeholder[target=b_bn1_num_batches_tracked]
    %x : [num_users=1] = placeholder[target=x]
    %conv2d : [num_users=1] = call_function[target=torch.ops.aten.conv2d.default](args = (%x, %p_conv1_weight, %p_conv1_bias, [1, 1], [1, 1]), kwargs = {})
    %_native_batch_norm_legit_no_training : [num_users=1] = call_function[target=torch.ops.aten._native_batch_norm_legit_no_training.default](args = (%conv2d, %p_bn1_weight, %p_bn1_bias, %b_bn1_running_mean, %b_bn1_running_var, 0.1, 1e-05), kwargs = {})
    %getitem : [num_users=1] = call_function[target=operator.getitem](args = (%_native_batch_norm_legit_no_training, 0), kwargs = {})
    return (getitem,)
```
